src/editors/containers/VideoEditor/components/VideoSettingsModal/__snapshots__/ErrorSummary.test.jsx.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ErrorSummary render snapshots: renders as expected when there are errors 1`] = `
+<Alert
+  show={true}
+  variant="danger"
+>
+  <Alert.Heading>
+    <FormattedMessage
+      defaultMessage="We couldn't add your video."
+      description="Title of validation error."
+      id="authoring.videoeditor.validate.error.title"
+    />
+  </Alert.Heading>
+  <p>
+    <FormattedMessage
+      defaultMessage="Please check your entries and try again."
+      description="Body of validation error."
+      id="authoring.videoeditor.validate.error.body"
+    />
+  </p>
+</Alert>
+`;
+
+exports[`ErrorSummary render snapshots: renders as expected when there are no errors 1`] = `
```

Comment:
Shouldn't this snapshot return nothing or at least the Alert param `show` should equal `false`

Reply:
I fixed this. Realized that the logic I had would have shown alert only if every widget had an error.